### ISSUE TITLE
Feature request: add title to pages

### ISSUE THREAD

**Issue by aawiseman** [fieldpapers/fieldpapers#190]
Feature request: the title of the atlas doesn't appear anywhere on the page -- I think that would be helpful. Otherwise you have to refer to the URL when talking about the atlas, which is confusing.

**Comment by mojodna** [comment]
Good idea!

**Comment by surabhibhargava** [comment]
@aawiseman  I would like to work on this but can't find any Developers guide/docs/wiki to understand the codebase. Can you help me get started with this? 
Thanks.

**Comment by ian-ross** [comment]
@surabhibhargava What have you actually read so far?  Have you got a development environment set up?

**Comment by surabhibhargava** [comment]
@ian-ross I started with this [wiki](http://wiki.openstreetmap.org/wiki/Field_Papers) to get a better idea of field papers. I went through [this](https://github.com/fieldpapers/fieldpapers) and read about the functioning of each module.

According to this page, for computer vision enthusiasts, I should refer to Field paper scanner, but in that file there was just the extract.cpp file which implements SURF feature detection. I looked at the other repos also. I think I am supposed to work on the fp-legacy repo but I'm not sure.

I started following [this](https://github.com/stamen/fieldpapers), later to realize that it is not the actual field papers repo when I got stuck.

Can you tell me how shall I go about installing the development setup?

Thanks.

**Comment by ian-ross** [comment]
There are some "getting started" instructions under one of the earlier project descriptions: https://github.com/Cadasta/cadasta-project-ideas/issues/2

The current image processing pipeline is in fp-legacy, as you say.

**Comment by mojodna** [comment]
I'll post some notices on other repos to try to avoid others running into the same thing. 

URLs are currently being output here: https://github.com/fieldpapers/fp-legacy/blob/modernize/decoder/compose.py#L372

**Comment by surabhibhargava** [comment]
@ian-ross I have installed the fp-web app setup mentioned under the getting started section. Initially I faced some issues with the versions of ruby and gem being different but then I changed the version of ruby mentioned in the gem file and now it works fine - I'm able to host the field papers page on my localhost.
What should be the next step to set up the image processing pipeline?
Also, for this task, which title are we talking about? Is it the name of the atlas that user gives?
